account_asset_management/models/account_asset.py

```diff
@@ -1285,13 +1285,16 @@ def _xls_acquisition_fields(self):
             "depreciation_base",
             "salvage_value",
         ]
+        if self.env.user.has_group("base.group_multi_company"):
+            acquisition_fields = acquisition_fields + ["company"]
```

Comment:
Why `company` and not `company_id`/`company_ids`?

Reply:
I use name `company` same as `account` field. (it not use account_id)
or should we change company to company_id?